apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/index.vue

```diff
@@ -212,6 +217,25 @@ function getSettingValue(settingKey: string, settings: ISetting[]) {
         }}
         cross-sectional.
       </LandingCardSecondary>
+
+      <LandingCardSecondary icon="viewTable">
+        <b>
+          {{ data.data.Subcohorts_agg.count }}
+          {{
+            getSettingValue(
+              "CATALOGUE_LANDING_DATASETS_LABEL",
```

Comment:
I am confused: datasets and subcohorts are not the same thing?

Reply:
see additional commits